app/queries/create_vimeo_video_mutator.rb

```diff
@@ -0,0 +1,34 @@
+class CreateVimeoVideoMutator < ApplicationQuery
+  include AuthorizeAuthor
+  
+  property :target_id, validates: { presence: true }
+  property :size, validates: { presence: true }
+  
+  def create_vimeo_video
+    path = '/me/videos'
+    data = {
+      upload: {
+        approach: 'tus',
+        size: size
+      },
+      privacy: {
+        embed: 'whitelist'
+      }
+    }
```

Comment:
I think this API implementation info can be moved into `Vimeo::ApiService`. The invocation (below) can be of the form:

```ruby
vimeo_api = Vimeo::ApiService.new(path, current_school)
response = vimeo_api.create_video(size)
```

and then have `Vimeo::ApiService` hold information about which path to use for which API _action_.